test/suites/scheduling/suite_test.go

```diff
@@ -131,6 +131,7 @@ var _ = Describe("Scheduling", Ordered, ContinueOnFailure, func() {
 				v1beta1.LabelSKUStoragePremiumCapable:     "true",
 				v1beta1.LabelSKUStorageEphemeralOSMaxSize: "53",
 				v1beta1.AKSLabelCluster:                   env.NodeResourceGroup,
+				v1beta1.AKSLabelScaleSetPriority:          "regular",
```

Comment:
We need to test on spot as well. I am not sure we ever actually write spot... just regular above. We likely need a place where we conditionally write spot onto the node too.